Pages/NewUserCreation.cshtml

```diff
@@ -0,0 +1,93 @@
+﻿@page
+@model CourseworkAC31007Agile.Pages.NewUserModel
+@using CourseworkAC31007Agile.BackEnd.DatabaseModel
+@using Microsoft.AspNetCore.Http
+
+@{
+    Layout = "_LayoutNewUser";
+}
+@{
+    if (HttpMethods.IsPost(Request.Method))
+    {
+        var fname = Request.Form["fname"];
+        var surname = Request.Form["surname"];
+        var role = Request.Form["role"];
+        var username = Request.Form["username"];
+        var pass1 = Request.Form["pass1"];
+
+        var user = new Users();
+        user.Username = username;
+        user.Password = pass1;
+        user.Name = fname + " " + surname;
+        user.PermissionLevel = role;
+
+        using (var db = new DatabaseContext())
+        {
+            var inserting = db.Users.Add(user);
+        }
```

Comment:
```suggestion
            db.SaveChanges();
        }
```
You need this to finalise any changes into the database. Without it, the database remains the same as before.